tests/testthat/test-derive-array.R

```diff
@@ -29,6 +29,16 @@ with_test_authentication({
         expect_identical(as.vector(ds$derivedarray[[1]]), q1.values)
     })
 
+    mr_derivation <- deriveArray(list(ds$q1, ds$petloc$petloc_home), selections=c("Dog"), name="Derived pets MR") # cat dog has id 2
+    ds$derivedmr <- mr_derivation
```

Comment:
If you're not going to assert anything about the shape of `mr_derivation`, just do the assignment into `ds` on the same line.